pygmt/src/histogram.py

```diff
@@ -0,0 +1,34 @@
+"""
+Histogram - Create a histogram
+"""
+from pygmt.clib import Session
+from pygmt.helpers import (
+    GMTTempFile,
+    build_arg_string,
+    fmt_docstring,
+    kwargs_to_strings,
+    use_alias,
+)
+
+
+@fmt_docstring
+@use_alias(
+    J="projection",
+    G="color",
+    R="region",
+)
+@kwargs_to_strings(R="sequence")
+def histogram(table, **kwargs):
```

Comment:
I believe you forgot `self`:
```suggestion
def histogram(self, table, **kwargs):
```

Reply:
Yep, I copied directly from grdinfo to get the virtual file example and forgot to include self from the other plotting functions. 🤦🏻‍♂️

Reply:
@seisman The use of `self` is causing a style check failure because it isn't used in the function; should I disable pylint for this error (follow-up question: how would I do that?)? The function itself appears to be working correctly, but needs `self` for it to work.